invenio_app_rdm/theme/templates/semantic-ui/invenio_app_rdm/landing_page/record_landing_page.html

```diff
@@ -25,8 +26,8 @@
   data-recid='{{ record["id"] | tojson }}'>
 </div>
 <div class="ui container">
-  <div class="ui padded relaxed grid">
-    <div class="two column row">
+  <div class="ui relaxed grid">
+    <div class="two column row top-padded">
```

Comment:
@mb-wali is this class coming from Semantic UI?

Reply:
no, not sure but, but i saw it here:https://github.com/inveniosoftware/invenio-app-rdm/blob/c793d20b56a0d3613d6ea1624add9839dd74db27/invenio_app_rdm/theme/assets/semantic-ui/less/invenio_app_rdm/landing_page.less#L22